test/gtest/diagnostics.cpp

```diff
@@ -141,20 +149,20 @@ TEST_F(Diagnostics, NestedCompileContexts) {
             @diagnostic("uninitialized_out_param", "disable")
         )"));
         EXPECT_TRUE(test);
-        EXPECT_EQ(0u, ::P4::diagnosticCount());
+        EXPECT_EQ(0u, diagnosticCount());
     }
 
     // Run a test with no diagnostic pragma for `uninitialized_out_param`. It
     // should default to triggering a warning; the diagnostic actions configured
     // by the previous tests should be gone.
     auto test = createP4_16DiagnosticsTestCase(P4_SOURCE(R"()"));
     EXPECT_TRUE(test);
-    EXPECT_EQ(1u, ::P4::diagnosticCount());
-    EXPECT_EQ(0u, ::P4::errorCount());
+    EXPECT_EQ(1u, diagnosticCount());
+    EXPECT_EQ(0u, errorCount());
```

Comment:
Nit: Is it better to also have a `EXPECT_EQ(1u, warningCount());` line here to be consistent with other test cases?